vite.config.ts

```diff
@@ -39,6 +39,8 @@ export default defineConfig({
 	},
 	build: {
 		sourcemap: 'inline' // helpful for debugging, maybe remove in production
-	},
-  global: {}
+	}, 
+	define: { 
+		'global.process':  JSON.stringify('window.process')
```

Comment:
Removing global here makes sense apparently but this line makes no sense and is not relevant for the fix.

```suggestion
```